plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -49,6 +49,7 @@ type SchedulerServer struct {
 	Address           util.IP
 	ClientConfig      client.Config
 	AlgorithmProvider string
+	AlgoProviders     string
```

Comment:
Please don't add this, I don't think you need it-- but if you did need it, I'd ask for it to be spelled out completely.